std/uni.d

```diff
@@ -8324,15 +8332,15 @@ auto asUpperCase(Range)(Range str)
     assert("hEllo".asUpperCase.equal("HELLO"));
 }
 
-// explicitly undocumented
+/// ditto
 auto asLowerCase(Range)(auto ref Range str)
```

Comment:
It's not exactly pretty have all of those overloads in the docs. If it weren't for the fact that the secondary overloads use `auto ref`, I'd argue that it should all just be merged into one function that used `static if`, but I don't think that we should normally be putting `auto ref` on ranges like that. So, maybe it should be documented, but it is kind of ugly to have a bunch of overloads like that which basically only differ by their template constraints.